colorization/src/model_process.py

```diff
@@ -59,24 +59,21 @@ def LoadModelFromFileWithMem(self, modelPath):
         if self.loadflag:
             logging.error("has already loaded a model")
             return 1
-        work_size, weight_size, ret = acl.mdl.query_size(modelPath, self.modelMemSize,
-                                                         self.modelWeightSize)
+        work_size, weight_size, ret = acl.mdl.query_size(modelPath)
         if ret != acl_constants.ACL_ERROR_None:
             logging.error("query model failed, model file is %s", modelPath)
             return 1
-        ret = acl.rt.malloc(self.modelMemPtr, self.modelMemSize,
-                            acl.ACL_MEM_MALLOC_HUGE_FIRST)
+        ret = acl.rt.malloc(self.modelMemSize, acl.ACL_MEM_MALLOC_HUGE_FIRST)
```

Comment:
values like acl_mem_malloc_huge_first are also contained in acl_constants.py, please check